reference/config_files/metadata.json.rst

```diff
@@ -0,0 +1,69 @@
+.. _metadata.json:
+
+metadata.json
```

Comment:
I don't think we should document this file at all.
This is internal implementation detail, subject to change at anytime

Reply:
I agree, it is the same for `registry.json` file. Are you ok removing both of them? I'd prefer to remove them, for sure.

Reply:
Yes, I am totally fine with that, I didn't realize, that shouldn't be documented either.

Maybe just the opposite: a note somewhere saying that these files are internal implementation details, subject to change at anytime, and "dont use them", and point to the remotes.txt in ``conan config install`` for the remote case.